docs/core/porting/libraries.md

```diff
@@ -13,336 +13,183 @@ ms.assetid: a0fd860d-d6b6-4659-b325-8a6e6f5fa4a1
-|Windows Phone|wpa|&rarr;|&rarr;|8.1|||||
-|Windows Phone Silverlight|wp|8.0|||||||
+[Porting to .NET Core - Analyzing your Third-Party Party Dependencies](~/docs/core/porting/third-party-deps.md)   
+This topic discusses the portability of third-party dependencies and what to do when a NuGet package dependency doesn't run on .NET Core.
 
-A key thing to understand is that **a project targeting a lower version cannot reference a project targeting a higher version**. For example, a project targeting the .NET Platform Standard version 1.2 cannot reference projects that target .NET Platform Standard version 1.3 or higher. Projects **can** reference lower versions, though, so a project targeting .NET Platform Standard 1.3 can reference a project targeting .NET Platform Standard 1.2 or lower.
+## .NET Framework technologies unavailable on .NET Core
 
-It's recommended that you pick the lowest possible .NET Standard version and use that throughout your project.
+Several technologies available to .NET Framework libraries aren't available for use with .NET Core, such as AppDomains, Remoting, Code Access Security (CAS), and Security Transparency. If your libraries rely on one or more of these technologies, consider the alternative approaches outlined below. For more information on API compatibility, the CoreFX team maintains a [List of behavioral changes/compat breaks and deprecated/legacy APIs](https://github.com/dotnet/corefx/wiki/ApiCompat) at GitHub.
 
-Read more in [.NET Platform Standard Library](../../standard/net-standard.md).
+Just because an API or technology isn't currently implemented doesn't imply it's intentionally unsupported. Feel free to file an issue in the [dotnet/corefx repository issues](https://github.com/dotnet/corefx/issues) at GitHub to ask for specific APIs and technologies. [Porting requests in the issues](https://github.com/dotnet/corefx/labels/port-to-core) are usually marked with the `port-to-core` label.
 
-## Key Technologies Not Yet Available on the .NET Standard or .NET Core
+### AppDomains
 
-You may be using some technologies available for the .NET Framework that are not currently available for .NET Core. Each of the following sub-sections corresponds to one of those technologies. Alternative options are listed if it is feasible for you to adopt them.
+AppDomains isolate apps from one another. AppDomains require runtime support and are generally quite expensive. They're not implemented in .NET Core. We don't plan on adding this capability in future.
 
-### App Domains
+For code isolation, we recommend separate processes or using containers as an alternative. For the dynamic loading of assemblies, we recommend the new <xref:System.Runtime.Loader.AssemblyLoadContext> class. Information (such as the name and base directory) is provided by APIs on other types, for instance `AppContext.BaseDirectory`. Some scenarios, such as getting the list of loaded assemblies, are unsupported, as they're inherently fragile.
```

Comment:
We don't have the ability to deal cleanly with versions just yet. I can add "in 1.0" and "for 2.0" language tho. I'm going to ping the CoreFX cats for a word on this, since this is really their content.